src/content/staking/solo/index.md

```diff
@@ -0,0 +1,175 @@
+  </ExpandableCard>
+  <ExpandableCard title="No withdrawing (for now)" eventCategory="SoloStaking" eventName="clicked no withdrawing">
+    Withdrawing staked ETH or rewards from a validator balance is not yet supported. Support for withdrawals are planned for the Shanghai upgrade following The Merge. You should anticipate your ETH being locked for at least one-to-two years. After Shanghai, this will no longer be the case, at which point you will be able to freely withdraw portions or all of your stake if you wish.
+  </ExpandableCard>
+  <ExpandableCard title="Maintenance" eventCategory="SoloStaking" eventName="clicked maintenance">
+    Hardware occasionally fails, connections error out, and client software occasionally needs upgrading. Node maintenance is inevitable and will occasionally require your attention. You'll want to be sure you stay aware of any anticipated network upgrades, or other critical client upgrades.
+  </ExpandableCard>
+  <ExpandableCard title="Reliable uptime" eventCategory="SoloStaking" eventName="clicked reliable uptime">
+    Your rewards are proportional to the time your validator is online and properly attesting. Downtime incurs penalties roughly equal to what you would have earned in that same amount of time. Being offline while the network is finalizing does not result in slashing.
+    Epoch rewards are inversely proportional to how long it takes your attestations to be included, meaning internet bandwidth also matter. This can vary, but a minimum of 10 Mb/s up and down is recommended.
+  </ExpandableCard>
+  <ExpandableCard title="Slashing risk" eventCategory="SoloStaking" eventName="clicked slashing risk">
+    Different from inactivity penalties for being offline, <em>slashing</em> is a much more serious penalty reserved for malicious offenses. By running a minority client with your keys loaded on only one machine at time, your risk of being slashed is minimized. That being said, all stakers must be aware of the risks of slashing.
+    
+    <a href="https://medium.com/prysmatic-labs/eth2-slashing-prevention-tips-f6faa5025f50/">More on slashing and validator lifecycle</a>
+  </ExpandableCard>
+</InfoGrid>
+
+<StakingComparison page="solo" />
+
+## How it works {#how-it-works}
+
+<HowSoloStakingWorks />
+
+If ever desired, you can exit as a validator which eliminates the requirement to be online, and stops any further rewards. Be aware that until the planned Shanghai upgrade _withdrawing_ those funds will not be possible.
+
+After Shanghai, users will be able to withdraw their rewards as well as their stake if they choose.
+
+## Get started on the Staking Launchpad {#get-started-on-the-staking-launchpad}
+
+The Staking Launchpad is an open source application that will help you become a validator. It walks you through a preparation checklist, how to choose clients, generate your keys and assists with depositing your ETH to the staking deposit contract.
```

Comment:
>  It walks you through a preparation checklist, how to choose clients, generate your keys and assists with depositing your ETH to the staking deposit contract.

This sentence feels a bit strange grammatically...

Reply:
Agree... will change to:

> The Staking Launchpad is an open source application that will help you become a staker. It will guide you through choosing your clients, generate your keys and depositing your ETH to the staking deposit contract. A checklist is provided to make sure you've covered everything to get your validator set up safely.